src/services/authenticator.js

```diff
@@ -75,10 +76,18 @@ function Authenticator() {
     });
   };
 
-  this.loginWithEmailOrTokenArgv = async (config) => {
-    const token = await this.login(config);
-    this.saveToken(token);
-    return null;
+  this.tryLogin = async (config) => {
```

Comment:
[Improvement] This doesn't return anything. We could probably return `true`/`false` to have an instant feedback about the success of the login in the code.

Reply:
I understand your point. In this case it's a procedure (i.e. not a function) with an imperative name (and behavior) so I guess it execute a sequence of orders. In other word, as a user of this procedure, I'm not sure what a `trySomething` return value would mean (but I would know what a `isSomething` will return a boolean and have no side effect). **Not sure about my opinion though!**

Reply:
I understand your point. In this case it's a procedure (i.e. not a function) with an imperative name (and behavior) so I guess it execute a sequence of orders. In other word, as a user of this procedure, I'm not sure what a `trySomething` return value would mean (but I would know what a `isSomething` will return a boolean and have no side effect). **Not sure about my opinion though!**

Reply:
the result of this method is: throw something or success.
I can set 'return true' instead of 'return null', but there is no significance.
I would prefer to return nothing but lint does not want it :)